delivery_schenker/models/delivery_carrier.py

```diff
@@ -359,11 +359,36 @@ def _schenker_shipping_information_package_volume(self, picking, package):
         )
 
     def _schenker_shipping_information_round_weight(self, weight, precision_digits=2):
-        return float_round(weight, precision_digits=precision_digits)
+        """
+        The schenker api requires 2 decimal points.
+        Round up to avoid 0 if <0.005.
+        round -> repr -> float to avoid extra decimals that error with the Schema.
+        float_round(2.086, precision_digits=2) = 2.0100000000000002
+        """
+        return float(
+            float_repr(
+                float_round(
+                    weight, precision_digits=precision_digits, rounding_method="UP"
+                ),
+                precision_digits=precision_digits,
+            )
+        )
 
     def _schenker_shipping_information_round_volume(self, volume, precision_digits=2):
-        """The schenker api requires 2 decimal points"""
-        return float_round(volume, precision_digits=precision_digits)
+        """
+        The schenker api requires 2 decimal points.
+        Round up to avoid 0 if <0.005.
+        round -> repr -> float to avoid extra decimals that error with the Schema.
+        float_round(2.086, precision_digits=2) = 2.0100000000000002
+        """
+        return float(
+            float_repr(
+                float_round(
+                    volume, precision_digits=precision_digits, rounding_method="UP"
+                ),
+                precision_digits=precision_digits,
+            )
+        )
```

Comment:
Could you create a own method for this?
Something like
"def _float_round"?

Reply:
@mt-software-de done